server/server.go

```diff
@@ -1010,6 +1010,14 @@ func (s *Server) parsePublishParams(r *http.Request, m *message) (cache bool, fi
 			return false, false, "", "", false, errHTTPBadRequestActionsInvalid.Wrap(e.Error())
 		}
 	}
+	extrasStr := readParam(r, "x-extras")
```

Comment:
Probably add `extras` to this list too

Reply:
👍

I noticed in the docs that `extras` are already an option in Intent actions. Are you okay with having that term used here and there?